runtime/pack/dist/opt/matchit/doc/matchit.txt

```diff
@@ -175,6 +175,22 @@ fail to skip matching groups in comments and strings.  If the |filetype|
 mechanism is turned off, the |b:match_words| variable will probably not be
 defined automatically.
 
+2.1 Temporarily disable the matchit plugin	*matchit-disable* *:MatchDisable*
```

Comment:
The "To use the matchit plugin after startup" paragraph above this could be changed to:

```
To use the matchit plugin after Vim has started, execute this command: >
       packadd matchit
```

to keep this synced, if you'd like